deployments/helm-chart/values.yaml

```
@@ -220,6 +223,9 @@ controller:
     ## Autogenerated if not set or set to "".
     # leaderElectionLockName: "nginx-ingress-leader-election"
 
+    ## The annotations of the leader election configmap
```

Tiny nitpick, (for consistency with other doc strings):
```suggestion
    ## The annotations of the leader election configmap.
```